packages/aws-cdk/lib/config.ts

```diff
@@ -436,7 +424,7 @@ export class DynamicValue {
   public static fromInline(f: () => any): DynamicResult {
     return {
       dynamicType: 'function',
-      dynamicValue: f,
+      dynamicValue: f.toString(),
```

Comment:
This makes the config object structuredClone-able. Previously `toString()` was just called a little later.